client/src/app/components/navigation-bar/navigation-bar.component.html

```diff
@@ -0,0 +1,103 @@
+<!-- Sidebar Wrapper (Two-column Layout) -->
+<div class="flex h-full rounded-2xl mr-4 bg-gray-100 transition-all duration-300 ease-in-out relative" [ngClass]="{ 'w-16': !isExpanded(), 'w-48': isExpanded() }">
+  <!-- Left Section: Main Navigation -->
+  <div class="flex flex-col p-3 items-center justify-between w-full">
+    <!-- Helios Icon -->
+    <a routerLink="/" [pTooltip]="!isExpanded() ? 'Home' : undefined" class="flex items-center gap-2 hover:bg-gray-200 w-full">
+      <app-helios-icon size="2rem" class="rounded-xl w-8 cursor-pointer" />
```

Comment:
issue: the rounded border does not work 

![image](https://github.com/user-attachments/assets/4be15d2f-3d33-4547-a9dd-798f03b6bdae)